### PR TITLE
New collations are not be supported

### PR DESCRIPTION
**Pull request by shiyuhang0**
### What problem does this PR solve? 
https://github.com/pingcap/tispark/issues/2236

### What is changed and how it works?
Set `new_collations_enabled_on_first_bootstrap = false`  for IT
Add Limitations in readMe

### CODE REVIEW

**Comment by shiyuhang0** [comment]
@xuanyu66 @zhangyangyu @birdstorm PTAL

File: README.md [hunk 1]
```diff
@@ -269,7 +269,10 @@ For more details about the test, see [here](./core/src/test/Readme.md).
 
 4. `Null in aggregration` is not supported, e.g. `select sum(null) from t group by col1`.
 
-5.  The dependency `tispark-assembly` should not be packaged into `JAR of JARS` file (for example, build with spring-boot-maven-plugin), or you will get `ClassNotFoundException`. You can solve it by adding `spark-wrapper-spark-version` in your dependency or constructing another forms of jar file.
+5. The dependency `tispark-assembly` should not be packaged into `JAR of JARS` file (for example, build with spring-boot-maven-plugin), or you will get `ClassNotFoundException`. You can solve it by adding `spark-wrapper-spark-version` in your dependency or constructing another forms of jar file.
+
+6. Tispark don't support collations now. If you use TiDB version > 5.4.0 or master, you need to change the config `new_collations_enabled_on_first_bootstrap` to false, for the defalut value change to `true` from `false`.
```
Comment (zhangyangyu) [inline README.md:274]: And for current TiDB versions(<= 5.4), if I change the value from default `false` to `true`, TiSpark will have problem too, right?

Comment (shiyuhang0) [reply README.md]: PTAL the new commit

**Review by zhangyangyu — APPROVED** [review]

**Review by sunxiaoguang — APPROVED** [review]
LGTM

**Review by xuanyu66 — APPROVED** [review]

**Pull request merged**